src/main/java/dev/waterdog/waterdogpe/WaterdogPE.java

```diff
@@ -43,6 +43,7 @@ public static void main(String[] args) {
         logger.info("§bStarting WaterDogPE proxy software!");
         logger.info("§3Software Version: " + versionInfo.baseVersion());
         logger.info("§3Build Version: " + versionInfo.buildVersion());
+        logger.info("§3Metrics publisher ID: " + versionInfo.metricsId());
```

Comment:
I don't think this is ever going to change, so we don't need to expose it in a log.